modules/command/src/main/java/org/apache/fluo/command/FluoExec.java

```diff
@@ -44,25 +50,39 @@ protected void configure() {
     }
   }
 
-  public static void main(String[] args) throws Exception {
-    if (args.length < 2) {
-      System.err.println("Usage: fluo exec <app> <class> args...");
-      System.exit(-1);
+  @Parameter(description = "<app> <class> args...", variableArity = true)
```

Comment:
Another quick note about this class, unlike the other commands it parses out the application name by position not by the -a option. Also it does not have a -o option for overriding configuration. It makes it slightly harder to reuse functionality and slightly breaks the consistency of the other commands. May not be something we address in this PR, but just wanted to throw that out there.